### PR TITLE
start wiring up executive summary

### PR DESCRIPTION
**Pull request by brendansudol**
### This pull request...
- adds first cut of executive summary section (still needs total amount across years [i think i can utilize the new budget reducer hotness] and better styling)

preview:
https://cl.ly/rzPx

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author

### This feature is done when...
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/containers/ExecutiveSummary.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const ExecutiveSummary = ({ data, years }) => (
+  <Section id="executive-summary" resource="executiveSummary">
+    <Subsection resource="executiveSummary.summary">
+      {data.map((d, i) => (
+        <div
+          key={d.id}
+          className="mb2 p2 sm-flex items-center alert alert-success"
+        >
+          <div className="p1 sm-m0 flex-auto">
+            {d.id !== 'all' && <div className="h5">Activity {i + 1}</div>}
```
Comment (mgwalker) [inline web/src/containers/ExecutiveSummary.js]: This `Activity {i + 1}` needs to go into the resource file, along with the `FFY {year}` below.

Comment (mgwalker) [reply web/src/containers/ExecutiveSummary.js]: Re: activity name, I think we already have a resource entry for that.  Maybe we should use that one?  Then it's consistent across locations?  Feels like a @quarterback question.

**Comment by NAretakis** [comment]
@brendansudol can we include the short activity description to this PR so that it displays for each activity?

image

**Comment by mgwalker** [comment]
✅ from code review perspective

**Comment by NAretakis** [comment]
@brendansudol can we add total cost for each row too? Basically, just summing the FFYs identified.

**Pull request merged**